python/lsst/ap/association/diaPipe.py

```diff
@@ -580,7 +607,9 @@ def run(self,
                                    doRunForcedMeasurement=self.config.doRunForcedMeasurement,
                                    )
 
-        return pipeBase.Struct(apdbMarker=self.config.apdb.value,
+        # For historical reasons, apdbMarker is a Config even if it's not meant to be read
```

Comment:
Minor suggestion for clarity:
`# For historical reasons, apdbMarker is a (empty) Config even if it's not meant to be read`

Reply:
I don't see anything historical about emptiness. Maybe "apdbMarker is of storage class Config"?

Reply:
OK, my suggestion was inaccurate. I think it looks surprising to return an empty config, so my request is to add context to this comment explaining why an empty `pexConfig.Config()` is returned.

Reply:
Then I'm not sure I understand the question. If `apdbMarker` is a Config, then you must provide _some_ config... or are you just looking for something obvious like "`Config()` is the cheapest valid dataset"?

Reply:
Yes, that's all. Just some clarifying comment about why we are returning an empty config even though everything ran successfully.